support/crown/.vscode/settings.json

```diff
@@ -0,0 +1,10 @@
+{
+    "rust-analyzer.check.command": "clippy",
+    "rust-analyzer.rustc.source": "discover",
+    "rust-analyzer.rustfmt.extraArgs": [
+        "--config", "unstable_features=true",
+        "--config", "binop_separator=Back",
+        "--config", "imports_granularity=Module",
+        "--config", "group_imports=StdExternalCrate"
+    ]
+}
```

Comment:
I think it doesn't make sense to check in this file as it can also container user-specific settings.

Reply:
I wonder if I can solve this with workspaces.

Reply:
we can